include/zephyr/bluetooth/bluetooth.h

```diff
@@ -691,10 +695,13 @@ enum {
 	BT_LE_ADV_OPT_DISABLE_CHAN_39 = BIT(17),
 
 	/**
+	 * \deprecated This option will be removed in the near future, see
+	 * https://github.com/zephyrproject-rtos/zephyr/issues/71686
+	 *
 	 * @brief Put GAP device name into advert data
 	 *
-	 * Will place the GAP device name into the advertising data rather
-	 * than the scan response data.
+	 * Will place the GAP device name into the advertising data rather than
+	 * the scan response data.
```

Comment:
This change seems unnecessary

Reply:
My formatter did it, I don't think that's a big issue. Especially knowing that those options will be deleted in the future :smile: